src/ConnectedMode/ServerConnectionsRepositoryAdapter.cs

```diff
@@ -77,18 +78,13 @@ private static ServerConnection MapConnection(Connection connection)
 
     private static ICredentials MapCredentials(ICredentialsModel credentialsModel)
     {
-        var securePassword = new SecureString();
         switch (credentialsModel)
         {
             case TokenCredentialsModel tokenCredentialsModel:
-                return new BasicAuthCredentials(tokenCredentialsModel.Token, securePassword);
+                return new BasicAuthCredentials(tokenCredentialsModel.Token.ToUnsecureString(), new SecureString());
```

Comment:
The token takes the place of the username. Is it on purpose a secure string?

Reply:
Yes, it was on purpose. Both the Token and password are secure strings. As Georgii pointed out, that is not for security reasons, but for compatibility. For consistency reasons, I thought the Token also makes sense. Should I invest the effort to revert it and then have the doubt in the future why one is secure and one not?